scripts/SANS/SANSadd2.py

```
@@ -86,45 +86,43 @@ def add_runs(  # noqa: C901
+            if is_not_allowed_instrument and is_first_dataset_event:
+                _report_error(f"Adding event data not support for {inst} for now")
+                _delete_workspaces([ADD_FILES_SUM_TEMPORARY, ADD_FILES_SUM_TEMPORARY_MONITORS])
                 return ""
 
-            for i in range(len(runs) - 1):
-                userEntry = runs[i + 1]
-                lastPath, lastFile, logFile, dummy, is_data_set_event = _load_ws(
-                    userEntry, defType, inst, ADD_FILES_NEW_TEMPORARY, rawTypes, period
+            for run_idx in range(1, len(runs)):
+                user_entry = runs[run_idx]
+                last_path, last_file, log_file, dummy, is_data_set_event = _load_ws(
```

Ruff (0.16.1)

[warning] Unpacked variable `dummy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)